test/sequential/test-module-loading.js

```diff
@@ -302,17 +302,14 @@ assert.throws(
       }
     },
     'fixtures/path.js': {},
-    'fixtures/throws_error.js': {},
```

Comment:
Unit test for module loading, verify module.children contains all the different modules that we've required,  **but should not contain modules that failed to load**